mk/spksrc.service.mk

```diff
@@ -195,6 +235,12 @@ ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
 		'."data-share" = {"shares": [{"name": $$share, "permission":{"rw":[$$user]}} ] }' $@ | sponge $@
 endif
 endif
+ifneq ($(strip $(SERVICE_WIZARD_SHARE_RESOURCE)),)
```

Comment:
Since this does the same thing maybe a boolean (USE_NEW_SHARE_WORKER=true or OLD_SHARE=false) is a better way to deal with this? e.g  `if  TCVERSION >= 7.0 || USE_NEW_SHARE_WORKER`

Reply:
done with USE_DATA_SHARE_WORKER = yes

Reply:
Thanks, that is also a much better name :+1: